### PR TITLE
Fix broken off-season link

### PR DESCRIPTION
**Pull request by aspenflow**

### CODE REVIEW

**Comment by aspenflow** [comment]
#2985

File: README.md [hunk 1]
```diff
@@ -52,7 +52,7 @@ We're back! Use this repo to share and keep track of software, tech, CS, PM, qua
  - 🇺🇸 - Requires U.S. Citizenship
  - 🔒 - Internship application is closed
 
-> For off-season internships, please see the [Off-Season 2024 README](./README-Off-Season.md).
+> For off-season internships, please see the [Off-Season 2024 README](https://github.com/SimplifyJobs/Summer2024-Internships/blob/dev/README-Off-Season.md).
```
Comment (sharunkumar) [inline README.md:55]: I think changing it to `(README-Off-Season.md)`  should work :thinking:

**Review by sharunkumar — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (sharunkumar) [reply README.md:55]: Strange, this used to work: https://github.blog/2013-01-31-relative-links-in-markup-files/

Maybe something is broken on github's side

**Comment by Rbeautelus1** [comment]
STOP

**Pull request closed without merging**